_meta/fields.common.yml

```diff
@@ -20,6 +20,90 @@
           description: >
             Timestamp of the event in microseconds since Unix epoch.
 
+    - name: url
+      type: group
+      description: >
+        A complete Url, with scheme, host and path.
+      fields:
+
+       - name: scheme
+         type: keyword
+         description: >
+           The protocol of the request, e.g. "https:".
```

Comment:
the example should be "https" - ecs is surprisingly specific on this - https://github.com/elastic/ecs#-url-fields.